### ISSUE TITLE
ThaumCraft Aspects not rendering on the Request UIs

### ISSUE THREAD

**Issue by Bernasss12** [RS485/LogisticsPipes#1224]
While on the Request Pipe's GUI when you shift hover over an item it shows the extra tooltip but it seems to add two extra empty lines, easier to show than to tell:
-Diamond block has no extra tooltip information so if only shows two empty lines:
![image](https://user-images.githubusercontent.com/6233500/58491127-5721f000-8166-11e9-81b0-8d1a3129ce5f.png)
-Provider Logistics Pipes has information to show but also has two empty lines after said information is shown:
![image](https://user-images.githubusercontent.com/6233500/58491208-8173ad80-8166-11e9-93ad-9c15a460c15d.png)

**Comment by hron84** [comment]
A random idea, but it seems like you lacking to use a mod that adds the mod name to items, like HWYLA or Mod Name Tooltip. I bet this causes the empty lines for you. Could you try out one of them and check if the empty lines are still present?

Also could you please include LP and Forge versions into your issue report? There is two different version of LP (one for 1.7.10 one for 1.12.2) and based on a screenshot it's quite hard to determine what version is affected in this bug.

**Comment by Bernasss12** [comment]
I could and maybe will indeed check if that makes any difference
The version I'm using is the dev version, I mainly created this issue to remind myself (and others) to take a look at this

**Comment by hron84** [comment]
It's cool to make reminders but I bet you will totally forget which versions did you use after a few weeks. Also if someone want to jump into this issue, he/she would want to re-create the environment to reproduce the issue, versions are important for this. Maybe you can get a solution earlier than you could get a time for fix it! 😄

**Comment by bziemons** [comment]
Well, if he wasn't also developing, I would totally agree @hron84 :) But I basically told him to make an issue, if he thinks it is an issue (instead of adding a TODO comment in the source code). Saying whether this occurs on 1.12 or 1.7.10 isn't necessary anymore. We are working on 1.12 only.

**Comment by davboecki** [comment]
@Bernasss12 Is Thaumcraft installed when the additional two lines appear?

**Comment by Bernasss12** [comment]
Yes it was, I was using the development environment. And I was already pretty sure the problem wasn't from logistics pipes since I tried to find it with no luck. Now that you've mentioned thaumcraft I'm sure this isn't LP's problem and the issue can be closed

**Comment by davboecki** [comment]
I think this is not really a problem but rather a side effect or missing interaction depending on how you look at it. I think it is Thaumcraft adding two empty lines to that they later can render their aspects over the tooltip to make it look like they are inside the tooltip.

**Comment by Bernasss12** [comment]
I had no idea that was the way the made that but that's actually pretty
smart, and yes in that case it's not a big at all even if the aspects are
not being shown, I guess now I will try to make them show instead of
removing the empty lines.

**Comment by bziemons** [comment]
What's the status on this @Bernasss12 ? I haven't been able to reproduce the issue.

**Comment by Bernasss12** [comment]
*I can still reproduce the bug in the most up-to-date branch:*

**In the Request Table the bug only happens in half the slots:**
<details><summary>Request Table Images</summary>
<p>

_Right side of the table_
![image](https://user-images.githubusercontent.com/6233500/61256976-e743e500-a766-11e9-806d-ebbd0f357eff.png)
_Left side of the table_
![image](https://user-images.githubusercontent.com/6233500/61256986-f2971080-a766-11e9-9162-9c7ed30a0284.png)
_Table with slots that work outlined green , and the ones that don't outlines red_
![image](https://user-images.githubusercontent.com/6233500/61257026-25d99f80-a767-11e9-9fbc-f39ed48d3a3a.png)

</p>
</details>

**The Request Pipe's GUI has the problem in every slot:**
<details><summary>Request Pipe Images</summary>
<p>

_Request Pipe MK1_
![image](https://user-images.githubusercontent.com/6233500/61257151-b1533080-a767-11e9-9911-48c91e3fe82d.png)
_Request Pipe MK2_
![image](https://user-images.githubusercontent.com/6233500/61257149-adbfa980-a767-11e9-99de-821e438cfdc4.png)

</p>
</details>

**The Add Macro Popup has the same problem:**
<details><summary>Request Table Add Macro Images</summary>
<p>

_Add Macro Popup_
![image](https://user-images.githubusercontent.com/6233500/61258190-47895580-a76c-11e9-803d-c74c343ed779.png)

</p>
</details>

**The statistics table, while choosing which item to track:**
<details><summary>Statistics Table Images</summary>
<p>

_Gui Statistics_
![image](https://user-images.githubusercontent.com/6233500/61258182-40fade00-a76c-11e9-8499-d05e1124fdf2.png)

</p>
</details>

## Possible solution
**While investigating and trying to solve the issue I got to the conclusion that the way LP is drawing the tooltips is going over one of the Events that the normal tooltips go through, and that Event is what triggers the default ThaumCraft Aspect Rendering**
Iirc the Event was called something along the lines `RenderToolTipEvent.PostText`

The problem I encountered while trying to find a solution myself was that a weird rendering clipping was happening, the aspects where being overlapped by the drawn items behind them: 
<details><summary>Statistics Table Images</summary>
<p>

_Don't know how well you can see that but here it is_
![image](https://user-images.githubusercontent.com/6233500/61259026-3f331980-a770-11e9-89c4-84fb3a99a770.png)

</p>
</details>

**Comment by Bernasss12** [comment]
I also checked with Logistics Pipes  version `LP0.10.2.195` with Thaumcraft version `6.1.BETA26`
